hw/ip/prim/pre_dv/prim_ascon/ascon_p_tb/README.md

```diff
@@ -0,0 +1,30 @@
+Ascon Permutation Verilator Testbench
```

Comment:
Do you also want to switch permutation to round here? And basically in all the files added with this commit? I think it would make sense and doing the change now is much easier compared to afterwards when you also add a TB for the permutation :-)

Reply:
good point. I pushed another commit, if you're fine with it, I'll squash it afterwards before merging

Reply:
This looks good, thanks @m-temp !